react-friends/src/PeopleList.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const PeopleList = props => {
+  return <div>{props.people.map(p => <div>{p.first_name} {p.last_name}:   {p.email}</div>)}</div>;
```

Comment:
I threw this on Abraham's PR as well but I figure it would be beneficial for you to see as well:

When you iterate through something (like using a map) you want to make sure that you add a key element to make sure each bit is unique or you'll get an error like Warning: Each child in an array or iterator should have a unique "key" prop

In this case, I believe you could write <div key={p.id}> or similar and it'd get rid of that error, since the id for each element is unique.